config/canonical.json

```diff
@@ -32613,11 +32613,13 @@
       "shop": "travel_agency"
     }
   },
-  "shop/travel_agency|Thomson": {
+  "shop/travel_agency|Thompson": {
```

Comment:
Correcting the name here does not use that much. If somebody builds the files again using `npm run build` the same name as before is added again.

Reply:
Oh, I didn't notice, sorry!